scarb/src/bin/scarb/args.rs

```diff
@@ -604,6 +609,30 @@ pub struct CompletionsArgs {
     pub shell: Option<Shell>,
 }
 
+/// Arguments accepted by the `tree` command.
+#[derive(Parser, Clone, Debug)]
+pub struct TreeCommandArgs {
```

Comment:
I've ignored the `--invert` flag (that `cargo tree` has) because it would employ a entirely different tree building algorithm -- and I don't need this feature for immediate use cases.

Same reason for `--edges`, though that one should be simpler to implement.